src/display/font_loader.js

```diff
@@ -71,13 +79,61 @@ class FontLoader {
     }
   }
 
+  async loadSystemFont(info) {
+    assert(
+      !this.disableFontFace,
+      "loadSystemFont shouldn't be called when `disableFontFace` is set."
+    );
+
+    const { loadedName, src, style } = info;
+    if (this.#systemFonts.has(src)) {
+      return;
+    }
+    this.#systemFonts.add(src);
+    if (this.isFontLoadingAPISupported) {
+      const fontFace = new FontFace(loadedName, src, style);
+      this.addNativeFontFace(fontFace);
```

Comment:
Doesn't this call already add the font to the document?
Can't we simply `await fontFace.loaded;` instead just below, similar to what we do elsewhere in this file?

Reply:
It doesn't appear that this suggestion was included in the changes; was it simply incorrect on my part?

Reply:
I thought `await fontFace.load()` was equivalent.
I replaced the `load()` by `loaded` and it just doesn't work.
In reading the `FontFaceSet::add` [specs](https://w3c.github.io/csswg-drafts/css-font-loading/#dom-fontfaceset-add) I don't see anything telling that the font must be loaded when added to the set.
So I'd say that we should call `load()` here and in `bind` function.
Wdyt ?

Reply:
Strange, since [our existing code](https://github.com/mozilla/pdf.js/blob/797f8d3dca5e0770c83788b3b65387884031caa0/src/display/font_loader.js#L82-L86) clearly works as-is.
Maybe there's a difference between supplying binary font-data vs a URL to a font-file?

Reply:
When we don't have the buffer, the data need to be fetched and it's fundamentally an async operation.
So my feeling is that it makes sense to have such a difference.

Reply:
Sounds reasonable, I mostly wanted to understand why there's a difference; thank you!